package.json

```diff
@@ -2,20 +2,24 @@
     "name": "wasm-xlsxwriter",
     "version": "0.1.8",
     "type": "module",
-    "main": "rust/pkg/wasm_xlsxwriter.js",
+    "main": "rust/pkg/nodejs/wasm_xlsxwriter.js",
+    "browser": "rust/pkg/web/wasm_xlsxwriter.js",
     "types": "rust/pkg/wasm_xlsxwriter.d.ts",
     "files": [
-        "rust/pkg/wasm_xlsxwriter_bg.wasm",
-        "rust/pkg/wasm_xlsxwriter.js",
-        "rust/pkg/wasm_xlsxwriter.d.ts"
+        "rust/pkg/web/wasm_xlsxwriter_bg.wasm",
+        "rust/pkg/web/wasm_xlsxwriter.js",
+        "rust/pkg/web/wasm_xlsxwriter.d.ts",
+        "rust/pkg/nodejs/wasm_xlsxwriter_bg.wasm",
+        "rust/pkg/nodejs/wasm_xlsxwriter.js",
+        "rust/pkg/nodejs/wasm_xlsxwriter.d.ts"
     ],
     "license": "MIT",
     "repository": {
         "type": "git",
         "url": "https://github.com/estie-inc/wasm-xlsxwriter"
     },
     "scripts": {
-        "build": "RUSTFLAGS='-C target-feature=+bulk-memory' wasm-pack build ./rust --target web --release && rm rust/pkg/.gitignore",
+        "build": "RUSTFLAGS='-C target-feature=+bulk-memory' wasm-pack build ./rust --target nodejs --out-dir pkg/nodejs --release && wasm-pack build ./rust --target web --out-dir pkg/web --release && rm rust/pkg/web/.gitignore && rm rust/pkg/nodejs/.gitignore",
```

Comment:
This should be separate script commands

Reply:
I guess only you have access to the github workflows? Maybe it's better you do the separation after the PR has been merged?